docs/enterprise/concepts.md

```diff
@@ -0,0 +1,140 @@
+
+Namespaces enable:
+
+- Self-Service.
+- Hierarchical policy enforcement (both enforced, and optional),
+- Policy organization.
+- RBAC for the Enterprise Console itself.
+
+Each of these sub-concepts are related and build on each other to form a unified security model.
+
+### Self-Service Capabilities
+
+One of the benefits of an identity-aware access proxy is that, once in place, developers and owners of enterprise applications have an incentive to configure their services to be accessible via the proxy.
+
+Self-service has [several benefits](https://www.usenix.org/system/files/login/articles/login_winter16_05_cittadini.pdf):
+
+- Frees global administrators from continuously modifying the configuration per user requests
+- Encourages service owners to own their configuration fragment
+- Ensures a reasonable compromise between development velocity and security controls
+
+Unlike with a VPN, or network driven access control mechanisms, application owners (with limited access permissions managed through namespaces) can maintain route and policy configuration for their own services, while  higher level operations, security, and identity teams are able to enforce higher level authorization and access policies.
+
+### Hierarchical Policy Enforcement
+
+Hierarchical policy lets administrators enforce high level authorization policy. Policies can be optional (self-select), or mandatory.
+
+Identity and access is organized and controlled with Identity and Access Management (**IAM**) teams via your Identity Provider (**IdP**). These users and groups are read by Pomerium from your IdP, and used to determine top-level Namespace organization.
+
+Consider this scenario: your organization has a security team managing high-level, course grain authorization controls. For example, they want to ensure that everyone with access to internal resources:
+
+   - has a `yourcompany.com` email account,
+   - isn't coming from a known bad actor IP address,
+
+From there, the security team delegates management of child [Namespaces](#namespaces) to application teams, providing flexibility to self-manage their own application [Routes](#routes) and [Policies](#policies).
+
+For example, a developer group can be given control to determine who has access to their Namespace, and create or edit Routes within it. They can provide authentication to their WiP app without writing new authorization code. Meanwhile, the CFO is given [manager](#manager) permissions over the "Accounting" Namespace, and can set enforced or optional policies for the services within.
+
+### RBAC for Enterprise Console Users
+
+- Namespaces are also used to achieve Role Based Access Control (**RBAC**) in the console itself.
+- There are three different roles:
```

Comment:
```suggestion
- There are four different roles:
```
 I guess it depends how you count Gust.

Reply:
When @travisgroth and I discussed it, we didn't consider Guest a role, but rather how a way to describe a user with no role.